Bottom: 327b4033a16c2fef165bad34b48ee66cd4e0f270
Top:    54b58c769ed3881babaf20863e5b2ceaf3ae3432
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 17:06:27 -0500

Refresh of update-ptxdist-build-factories

---
diff --git a/roles/buildbot-worker-ptxdist/templates/pip.conf.j2 b/roles/buildbot-worker-ptxdist/templates/pip.conf.j2
deleted file mode 100644
index a54ae09a..00000000
--- a/roles/buildbot-worker-ptxdist/templates/pip.conf.j2
+++ /dev/null
@@ -1,3 +0,0 @@
-[global]
-trusted-host = {{hostvars['buildbot-worker-devpi'].uri | urlsplit('hostname')}}
-index-url = {{hostvars['buildbot-worker-devpi'].uri}}
